src/richie/plugins/simple_picture/defaults.py

```diff
@@ -0,0 +1,50 @@
+"""Default settings for Richie's simple picture plugin."""
+from django.conf import settings
+
+SIMPLEPICTURE_PRESETS = getattr(
+    settings,
+    "RICHIE_SIMPLEPICTURE_PRESETS",
+    {
+        "default": {
+            "src": {"size": (500, 500), "crop": "smart"},  # easythumbnail options
+            "srcset": [
+                {  # list of tuples ({easythumbnail options}, {srcset description})
```

Comment:
I don't get your comment: what tuples are you pointing to?

Reply:
Fixed. This was a comment left from an earlier implementation.